pkg/api/defaults.go

```diff
@@ -774,7 +776,7 @@ func (p *Properties) setTelemetryProfileDefaults() {
 	}
 
 	if len(p.TelemetryProfile.ApplicationInsightsKey) == 0 {
-		p.TelemetryProfile.ApplicationInsightsKey = DefaultApplicationInsightsKey
+		p.TelemetryProfile.ApplicationInsightsKey = telemetry.AKSEngineAppInsightsKey
```

Comment:
Should this be empty string since TelemetryProfile.ApplicationInsightsKey is only used for user keys?